### PR TITLE
Build: Refine workflow for cygwin and remove scorecard. v6.0.120

### LINKED ISSUE #3982: GitHub action to set up Cygwin Failed to run
!!! Before submitting a new bug report, please ensure you have searched for any existing bugs and utilized 
the `Ask AI` feature at https://ossrs.io or https://ossrs.net (for users in China). Duplicate issues or 
questions that are overly simple or already addressed in the documentation will be removed without any 
response.

**Setup Cygwin Action Failed**
HTTP status 404 fetching http://mirrors.kernel.org/sourceware/cygwin/x86_64/setup.zst.sig
HTTP status 404 fetching http://mirrors.kernel.org/sourceware/cygwin/x86_64/setup.xz.sig
HTTP status 404 fetching http://mirrors.kernel.org/sourceware/cygwin/x86_64/setup.bz2.sig
HTTP status 404 fetching http://mirrors.kernel.org/sourceware/cygwin/x86_64/setup.ini.sig

Because the signatures are all merged to one file: 
http://mirrors.kernel.org/sourceware/cygwin/x86_64/sha512.sum

**Impact**
Github Workflow Setup Cygwin action.

**To Reproduce**
check 
https://github.com/ossrs/srs/actions/runs/8213483121/job/22464987462?pr=3981

**Expected behavior**
Github workflow can works well.

### PR DESCRIPTION
**Pull request by suzp1984**
#3983  already fixed the `test` workflow, but I think the `release` will have same issue.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -60,7 +60,6 @@ jobs:
           platform: x86_64
           packages: bash make gcc-g++ cmake automake patch pkg-config tcl unzip
           install-dir: C:\cygwin64
-          check-sig: false
```
Comment (winlinvip) [inline .github/workflows/test.yml:63]: Why remove this line?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -108,7 +108,7 @@ jobs:
       # See https://github.com/cygwin/cygwin-install-action#parameters
       # Note that https://github.com/egor-tensin/setup-cygwin fails to install packages.
       - name: Setup Cygwin
-        uses: cygwin/cygwin-install-action@db475590d56881c6cef7b3f96f6f3dd9532ea1f4 # master
+        uses: cygwin/cygwin-install-action@006ad0b0946ca6d0a3ea2d4437677fa767392401 # master
```
Comment (winlinvip) [inline .github/workflows/release.yml:111]: Without `check-sig: false` ?

Comment (suzp1984) [reply .github/workflows/release.yml:111]: http://mirrors.kernel.org/sourceware/cygwin/x86_64/

check above link.

Look, the mirrors updated on 17th-Mar, to fixed the signature missing problem.
So, I think the check-sig option can be opened by default.
The new PR https://github.com/ossrs/srs/pull/3995 just upgrade the release action to latest commit too.

File: .github/workflows/test.yml (hunk 1 above)
Comment (suzp1984) [reply .github/workflows/test.yml:63]: let the `cygwin-install` action do signature check. 
I once disable the signature check, to workaround #3982 , the cause is the sig file missing inside http://mirrors.kernel.org/sourceware/cygwin/x86_64/. But it seems the sig file come back, then open the sig check again.

**Pull request merged**